src/app/product-list.tsx

```diff
@@ -14,21 +15,13 @@ type ProductsListProps = {
   onEndReached: () => void;
 };
 
-const FiltersButton = ({
-  products,
-  query,
-}: {
-  products: Product[];
-  query: string;
-}) => {
+const FiltersButton = ({ products }: { products: Product[] }) => {
   if (products.length === 0) {
     return;
   }
   return (
     <View
-      className={`absolute ${
-        query === '' ? 'bottom-20' : 'bottom-28'
-      } flex w-auto items-center justify-center self-center pb-20`}
+      className={`absolute -bottom-8 flex w-auto items-center justify-center self-center pb-20`}
```

Comment:
since you are not using string interpolation you can just define it

```suggestion
      className="absolute -bottom-8 flex w-auto items-center justify-center self-center pb-20"
```